frontend/src/Constants.ts

```diff
@@ -0,0 +1,3 @@
+export const API_BASE: string | undefined = import.meta.env
```

Comment:
Die Konstanten in der Klasse werden im Code nicht verwendet und kann somit entfernt werden.